app/views/layouts/base.html.erb

```diff
@@ -32,26 +32,26 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
-    <%= javascript_include_tag *webpack_asset_paths('foreman-vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('bundle', :extension => 'js') %>
-    <%= javascript_include_tag 'application' %>
-    <%= webpacked_plugins_with_global_js %>
-    <%= webpack_dev_server %>
-    <%= yield(:javascripts) %>
-
-    <script type="text/javascript">
-      <%= yield(:inline_javascripts) %>
-    </script>
-
+    
     <%= yield(:head) %>
   </head>
 
   <body class="pf-m-redhat-font">
-    <!--[if lte IE 9]>
+    <!--[if IE]>
       <%= _('You are using an unsupported browser.') %>
+      <%= _('See list of supported browsers: %s') % 'https://theforeman.org/manuals/nightly/index.html#3.1.4BrowserCompatibility' %>
     <![endif]-->
     <% if User.current %>
+      <%= javascript_include_tag *webpack_asset_paths('foreman-vendor', :extension => 'js') %>
+      <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js') %>
+      <%= javascript_include_tag *webpack_asset_paths('bundle', :extension => 'js') %>
+      <%= javascript_include_tag 'application' %>
+      <%= webpacked_plugins_with_global_js %>
+      <%= webpack_dev_server %>
+      <%= yield(:javascripts) %>
+      <script type="text/javascript">
+        <%= yield(:inline_javascripts) %>
+      </script>
```

Comment:
Are these pages not needed for anonymous pages (such as the login page)? I'd think that you want to include them unconditionally and thus move it above `<% if User.current %>`.

Reply:
I don't think they are needed for the login page, do you see any usage for that?
in addition, some of those JS yields can contain code related to webpack when loading react components from plugin's .erb which means they need all of the above bundled scripts to exist.

Reply:
I'd think `webpacked_plugins_with_global_js` and `webpack_dev_server` are useful, since it gives you live reload while you're developing on the login page. As for the other pages: it probably doesn't make a difference, but keeping fewer code paths is better. Every if you have in your code means you technically need to do more testing so I'm always hesitant with it. Especially since you're breaking basic assumptions pages could rely on previously, potentially making a breaking change for plugins.

So at the very least, move the duplicate code that's in both branches outside of the condition but really I'd keep this as compatible as possible and move the existing code verbatim to the body (just as the PR description says).